secrets/values.yaml

```diff
@@ -0,0 +1,90 @@
+# secret main file
+
+# global values
+global:
+  auth: "oidc"
+  authIdToken: "FIXME"
+  authVO: "Rubin.production"
+
+  activemq:
+    serverList:
+      - "activemq-dev:61613"
+
+    basePasswd: "FIXME"
+
+  idds:
+    restHost: "idds-rest"
+    restPort: 8443
+
+  rucio:
+    rucioHost: "sprucio01.sdcc.bnl.gov:443"
+    authHost: "sprucio01.sdcc.bnl.gov:443"
+    username: "iddsv1"
+    password: "FIXME"
+    account: "iddsv1"
+
+  oidc:
+    panda_dev:
+      client_id: FIXME
+      client_secret: FIXME
+    rubin:
+      client_id: FIXME
+      client_secret: FIXME
+    rubin_production:
+      client_id: FIXME
+      client_secret: FIXME
+
+msgsvc:
+  enabled: true
+  name: "msgsvc-secret"
+
+panda:
+  enabled: true
+  name: "panda-secret"
+
+  database:
+    password: FIXME
+    dbhost: "FIXME"
+
+harvester:
+  enabled: true
+  name: "harvester-secret"
+
+  database:
+    password: FIXME
+    admin_password: FIXME
+    dbhost: "FIXME"
+
+idds:
+  enabled: true
+  name: "idds-secret"
+
+  database:
+    name: "idds_db"
+    user: "idds"
+    password: "MyPass123"
+    schema: "doma_idds"
+    host: "idds-dev-postgres"
+
+  activemq:
+    detination: /queue/idds_panda
```

Comment:
Do you really need to have these two lines in secret?

Reply:
It's not required. I added it here because for activemq, there are 5 environment variables for iDDS. 4 of them are in secrets. It would be convenient to add all of them to secrets.

Reply:
I don't see any usecase to change them. It is enough to put 
```
IDDS_CONDUCTOR_DESTINATION: "/queue/idds_panda"
IDDS_CONDUCTOR_USERNAME: "idds"
```
Generally, it is better if values.yaml contains only parameters the user has to change.

BTW, your comment has another implication. Do you need both confimap and secret to
define env variables for the instance? E.g.,

secrets/templates/panda.yaml
panda/templates/configmap.yaml
panda/charts/server/templates/configmap.yaml

they define env variables for panda server. Isn't it better to merge them to secrets/templates/panda.yaml, so we don't have to maintain multiple files? JEDI, server, and panda-database can use the same env 
var set.

Reply:
will work on this part to merge environments to secrets.
I am also testing how to adapt the auth_config to iDDS. In iDDS I have only one auth.cfg file which includes all clients. It's different to use one environment to include it (I tried different solutions this morning it doesn't work). I will created a file for it and update the iDDS docker to use it.

Reply:
I removed envs from different configmap files or just remove some configmap files.